lib/recurly/account.rb

```diff
@@ -42,6 +42,12 @@ class Account < Resource
     # @return [AccountBalance, nil]
     has_one :account_balance, readonly: true
 
+    # @return [Account, nil]
+    belongs_to :parent_account, class_name: "Account"
```

Comment:
We have a convention of using symbols for the account class name. Is there a reason we are using string here?

Reply:
No, I just didn't look at other uses of `belongs_to`. I'll change that to a symbol.